glustercli/cmd/volume.go

```diff
@@ -66,6 +71,9 @@ func init() {
 	// Volume Delete
 	volumeCmd.AddCommand(volumeDeleteCmd)
 
+	volumeGetCmd.Flags().BoolVar(&flagGetAdv, "advanced", false, "Get advanced options")
+	volumeGetCmd.Flags().BoolVar(&flagGetBsc, "basic", false, "Get basic options")
```

Comment:
Wouldn't `--basic` be the default ? In which case the value must default to `true`

Reply:
by default it will display all options, do you want me to show only ```default``` options?

Reply:
Yep. As mentioned by aravinda here: https://github.com/gluster/glusterd2/issues/503#issuecomment-353520736

Reply:
He also mentioned we need to display all options if ```--advanced``` flag is set. currently, I am listing only advance options if ```--advanced``` flag is set.

Reply:
if ```--advanced``` flag is set , will display both advance and basic options